### ISSUE TITLE
how easymotion index places

### ISSUE THREAD

**Issue by lnxfsf** [easymotion/vim-easymotion#496]
hi, i have question, how easymotion index places
and can that be changed somehow ?

Like, i noticed, when i use bidirectional 'f' (or 'f' or 't' in easymotion), first entry is 'a', but then, second and so on are inconsistent.. 
i don't know how this could be changed to go alphabeticall. like, to go 'a', 'b' 'c' etc.. because for my eyes, it's much more consistent. otherwise, with this current behaviour, i don't know what character it's gonna be on that place

**Comment by sud0xenn** [comment]
Exactly, I would like to tweak it so it's alphabetic like https://github.com/cirnovsky/vim-gotoword allows
